Cubical/Algebra/CommAlgebra/QuotientAlgebra.agda

```diff
@@ -186,21 +207,22 @@ module _ {R : CommRing ℓ} (A : CommAlgebra R ℓ) (I : IdealsIn A) where
                             (isPropIsCommIdeal (CommAlgebra→CommRing A))
                             refl ⟩
     _                  ≡⟨  CommRing.kernel≡I {R = CommAlgebra→CommRing A} I ⟩
-    I ∎
+    I                  ∎
 
 
 private
-  module _ (R : CommRing ℓ) where
+  module _ (R : CommRing ℓ) where abstract
```

Comment:
Is this 'abstract' needed for something? 'lemma' should be behind an abstract barrier anyway, right?

Reply:
No, that is not how the `abstract` keyword works (unfortunately). [From the documentation](https://agda.readthedocs.io/en/v2.6.2.2/language/abstract-definitions.html): "The reach of the `abstract` keyword does not extend into modules". I just double-checked that this is even the case if the inner module is anonymous.

Reply:
I moved the lemma, but left it marked abstract, because it seems more natural to me to have it as an abstract private definition than to have it as a non-abstract private definition. But this is only a slight preference, I can change it if you like it better the other way.

Reply:
I have no strong opinion on the inner abstract - just do what you like. But now that you have the outer module, you could move the implicit arguments of 'isZeroFromIdeal' to the outer module and use the correct ones in the lemma. Might look a bit impure, but I think it would be easier to read, if there aren't two different 'R's... But I want to leave it up to you, just let me know if you want to merge it like it is or make a change.

Reply:
Agreed. But now I finally saw how to avoid the lemma all together, which is I think even better. :-)